.github/workflows/test-python.yml

```diff
@@ -85,6 +85,7 @@ jobs:
       - name: Install the test dependencies
         run: |
           python -m pip install -v -e ".[test]"
+          python -m pip install -U --pre pandas
```

Comment:
should we be installing with a direct call to pip rather than requirements.txt?

Reply:
Removed